### DISCUSSION TITLE
A necessary tune in AQ

### DISCUSSION THREAD

**Discussion by Rholik** [chromiecraft/chromiecraft#3843] [General]
If you know about the server, the protection paladin is a thing and the emperor Vek'lor is inmune to physical damage. The fact the prot pala can agroo perfectly Vek'lor with consecration, hand and talent abilities makes really cheeseable making all the effort for shadow resist warlock worthless. So i suggest make Vek'lor not only inmune to physical damage but also holy damage. That avoids some cheese guilds speedruning To C'thun like nothing

**Comment by balleny** [comment]
holy pala tank was a valid tank strategy. no need to change that.
